util/log.py

```diff
@@ -6,10 +6,7 @@
 import sys
 import time
 
-try:
-    from dlstbx.util.colorstreamhandler import ColorStreamHandler
-except ImportError:
-    ColorStreamHandler = None
+from colorlog import ColoredFormatter
```

Comment:
We will need to trap the import similarly to the dlstbx version because even when we add the dependency everywhere not all development environments will pick it up automatically. Specifically we don't currently update conda environment dependencies.